heads or tails

```diff
@@ -0,0 +1,18 @@
+print('Игра "Орел-Решка"')
+print("Введите '0', если выбираете Орел")
+print("Введите '1', если выбираете Решка")
+user_input = input("Enter number: ")
+choice=" "
+if int(user_input) == 0:
+    choice = "Орел"
+else:
```

Comment:
it's better to add another check like elif. What if I input "2"? What if I input "ABC"?